pyproject.toml

```
@@ -2,14 +2,14 @@
 name = "shiver"
 description = "Spectroscopy Histogram Visualizer for Event Reduction"
 dynamic = ["version"]
-requires-python = ">=3.8"
 dependencies = [
     "mantidworkbench >= 6.6.20230517",
-    "pyoncat == 1.6.1"
+    "pyoncat ~= 1.6.1"
```

You're doing a fuzzy match with `~=`, drop the patch level
```suggestion
    "pyoncat ~= 1.6"
```

---

done!